tests/10apidoc/09synced.pl

```diff
@@ -393,7 +393,7 @@ sub assert_state_room_members_match {
 
    my @members = grep { $_->{type} eq 'm.room.member' } @{ $events };
    @members == scalar @{ $member_ids }
-      or die "Expected only ".(scalar @{ $member_ids })." membership events";
+      or die "Expected only ".(scalar @{ $member_ids })." membership events (got ".(scalar @members).")";
```

Comment:
for future reference, it's generally easier to replace this sort of construction with a call to `assert_eq`. (`assert_*` didn't exist very early in sytest's history, so this sort of `foo == bar or die` construction typically predates its existence)